control/content/assets/css/bf_base.css

```diff
@@ -0,0 +1,881 @@
+/* General */
+@font-face {
+  font-family: 'Conv_apercu_bold';
+  src: url('fonts/apercu_bold.eot');
```

Comment:
Why do we keep adding custom fonts? That should be inherited from buildfire right? Is there any specific reason?